### PR TITLE
Updated Venue info

### PR DESCRIPTION
**Pull request by BenWoodford76**
Got Bored so went through and updated Venue information

### CODE REVIEW

File: _venues/new-theatre-studio-a.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ Part of the Archaeology and Classics building left up when building was demolish
 Properly converted into a studio space in 2013, white to black walls, blue carpet to
 vinyl floor, rig and drapes installed.
 
+Studio A, with its slightly larger size and installed dimming is the main performace space for our Fringe season in the New Theatre building. However, many fringe shows explore other venues both on and off of University Park Campus
```
Comment (davidtaylorhq) [inline _venues/new-theatre-studio-a.md]: "Installed dimming" - non-techies won't understand it. Maybe "installed lighting system"?

File: _venues/new-theatre-studio-b.md [hunk 2]
```diff
@@ -13,4 +13,6 @@ Part of the Archaeology and Classics building left up when building was demolish
 Properly converted into a studio space in 2013, white to black walls, blue carpet to
 vinyl floor and rig installed. Drapes were installed later in 2015.
 
+Studio B is used as the Main rehearsal space within the New Theatre building.
```
Comment (davidtaylorhq) [inline _venues/new-theatre-studio-b.md]: as well as hosting occasional workshop sessions and fringe shows.

File: _venues/new-theatre.md [hunk 3]
```diff
@@ -17,3 +17,5 @@ location:
 
 Originally part of the Archaeology and Classics building, the main auditorium of the Nottingham New Theatre was given to the company in 1965. The foyer was remodeled in 2001 and again in 2012.
 
+The Main auditorium is a 80 seat venue when used for end-on productions, and is used by both In House and External companies. The venue is well maintained and the first two rows of seating can be removed for performances in Thrust or the Round.
```
Comment (davidtaylorhq) [inline _venues/new-theatre.md]: `end-on productions`
Technical term is proscenium I believe

Comment (davidtaylorhq) [inline _venues/new-theatre.md]: Also "Thrust" and "Round" probs don't need capital letters...?

File: _venues/trent-building.md [hunk 4]
```diff
@@ -7,4 +7,4 @@ location:
   lon: -1.196107
 ---
 
-First opened in 1928, Trent Building has been used by the university for mainly teaching and some administrative offices. 
\ No newline at end of file
+First opened in 1928, Trent Building has been used by the university for mainly teaching and some administrative offices. The Trent building is now the home of the Schools of English and Foreign Languages, with offices also for the Vice-Chancellor and other univeristy administration.
```
Comment (davidtaylorhq) [inline _venues/trent-building.md]: > and some administrative offices

> and other univeristy administration
Doesn't need repeating twice

File: _venues/new-theatre-studio-a.md (hunk 1 above)
Comment (BenWoodford76) [inline _venues/new-theatre-studio-a.md]: T'is fine :) finickity is good

File: _venues/trent-building.md (hunk 4 above)
Comment (BenWoodford76) [inline _venues/trent-building.md]: Ah, yes, my idiocy

**Comment by davidtaylorhq** [comment]
@nhyty @BenWoodford76 see my comments on the commit

https://github.com/newtheatre/history-project/commit/588bab9191cfee08f9c19dbb11dec317c4581c5f

**Comment by samozzy** [comment]
Found more spag. Soz.

**Pull request merged**